### PR TITLE
Brought Settings screen in ProfileView.

### PR DESCRIPTION
**Pull request by RomanEsin**

### CODE REVIEW

File: Project SF/Views/Tabs/Profile/ProfileView.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+//
+//  Created by Roman Esin on 13.07.2020.
+//
+
+import SwiftUI
+
+struct ProfileView: View {
+    var body: some View {
+        NavigationView {
+            Text("Profile View")
```
Comment (Priva28) [inline Project SF/Views/Tabs/Profile/ProfileView.swift:13]: Where is the profile name, username, pfp, etc?

**Review by Priva28 — CHANGES_REQUESTED** [review]

File: Project SF/Views/Tabs/Profile/ProfileView.swift (hunk 1 above)
Comment (Priva28) [reply Project SF/Views/Tabs/Profile/ProfileView.swift:13]: Otherwise its fine

Comment (RomanEsin) [reply Project SF/Views/Tabs/Profile/ProfileView.swift:13]: This PR is only for organising everything properly. The profile data is still to be added

Comment (Priva28) [reply Project SF/Views/Tabs/Profile/ProfileView.swift:13]: So you'll add it later?

Comment (RomanEsin) [reply Project SF/Views/Tabs/Profile/ProfileView.swift:13]: Yeah

Comment (Priva28) [reply Project SF/Views/Tabs/Profile/ProfileView.swift:13]: K

**Review by Priva28 — APPROVED** [review]

**Pull request merged**